pudb/debugger.py

```diff
@@ -2109,7 +2113,16 @@ def fallback(error_message):
                         else:
                             runner = shell.custom_shell_dict["pudb_shell"]
 
-                runner(curframe.f_globals, curframe.f_locals)
+                pythonstartup = os.getenv("PYTHONSTARTUP", os.devnull)
+                f_globals = deepcopy(curframe.f_globals)
+                f_locals = deepcopy(curframe.f_locals)
```

Comment:
That won't fly. We want to be able to retain the ability to modify locals. At the same time, it's not great to have `$PYTHONSTARTUP` modify the local frame. I'm not sure there's a good way forward here.

Reply:
So should we remove `deepcopy()`?

Reply:
Removed.

<https://github.com/inducer/pudb/discussions/580#discussioncomment-4801771> seems to be a better solution.